girder/api/sftp.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2016 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import paramiko
+import six
+import stat
+import time
+
+from girder import logger
+from girder.models.model_base import AccessException, ValidationException
+from girder.utility.path import lookUpPath, NotFoundException
+from girder.utility.model_importer import ModelImporter
+from six.moves import socketserver, StringIO
+
+DEFAULT_PORT = 8022
+DEFAULT_KEY = paramiko.RSAKey.from_private_key(StringIO("""-----BEGIN RSA PRIVATE KEY-----
```

Comment:
It makes me nervous to see a private key hanging out.  Can you add a docstring explaining why this is here and if it should be changed when deployed?